content/03-type-system.md

```diff
@@ -454,18 +454,18 @@ private abstract A { ... }
 
 > ##### Define: Private type
 >
-> A type can be made private by using the `private` modifier. As a result, the type can only be directly accessed from within the [module](define-module) it is defined in.
+> A type can be made private by using the `private` modifier. Afterwards, the type can only be directly accessed from within the [module](define-module) it is defined in.
 > 
-> Private types, unlike public ones, do not become a member of their containing package.
+> Private types, unlike public types, do not become a member of their containing package.
```

Comment:
Out of curiosity: Is the previous version wrong or is that just a style thing?

Reply:
Admittedly some changes might be a bit more of personal style leaking in, although I'm trying to stay consistent with the manual's existing approach to things. Nothing's explicitly wrong, but the change sounded better to me personally when read out loud.

Reply:
On second thought I'll revert this. I was treating this change as if it was discussing two unique objects, instead of one with multiple states.